Bloom/user.css

```diff
@@ -48,14 +55,16 @@
 }
 
 .main-home-homeHeader,
-.main-actionBarBackground-background {
+.main-actionBarBackground-background,
+.n87ifzfQlmGX8vkeYGDL {
```

Comment:
This class probably needs mapping in [spicetify-cli](https://github.com/spicetify/spicetify-cli), can you check there what it might be?